# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnome-libs14/remove_duplicate_definition.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

taken from fedora core 4

diff -urN gnome-libs-1.4.2.orig/gtk-xmhtml/colors.c gnome-libs-1.4.2/gtk-xmhtml/colors.c
--- gnome-libs-1.4.2.orig/gtk-xmhtml/colors.c	2002-08-06 23:25:41.000000000 +0200
+++ gnome-libs-1.4.2/gtk-xmhtml/colors.c	2005-09-05 13:57:38.410340500 +0200
@@ -292,33 +292,6 @@
 #define BMASK		0xc0
 #define BSHIFT		6
 
-/* XXX: This function does an XQueryColors() the hard way, because there is
- * no corresponding function in Gdk.
- */
-#ifndef WITH_MOTIF
-static void
-my_x_query_colors(GdkColormap *colormap,
-		  GdkColor    *colors,
-		  gint         ncolors)
-{
-	XColor *xcolors;
-	gint    i;
-
-	xcolors = g_new(XColor, ncolors);
-	for (i = 0; i < ncolors; i++)
-		xcolors[i].pixel = colors[i].pixel;
-
-	XQueryColors(gdk_display, GDK_COLORMAP_XCOLORMAP(colormap), xcolors, ncolors);
-
-	for (i = 0; i < ncolors; i++) {
-		colors[i].red   = xcolors[i].red;
-		colors[i].green = xcolors[i].green;
-		colors[i].blue  = xcolors[i].blue;
-	}
-
-	g_free(xcolors);
-}
-#endif
 /*****
 * Name: 		tryColor
 * Return Type: 	Boolean
